mozfun/event_analysis_udf/escape_metachars/udf.sql

```diff
@@ -0,0 +1,12 @@
+CREATE OR REPLACE FUNCTION event_analysis_udf.escape_metachars(s STRING)
```

Comment:
Should probably document this, but the reference for re2 is here: https://github.com/google/re2/wiki/Syntax